.github/workflows/build-java-multi.yaml

```diff
@@ -57,9 +59,23 @@ jobs:
           repository-ref: ${{ env.GITHUB_REF_NAME }}
           repository-access-token: ${{ secrets.REPO_ACCESS_PAT }}
 
+      - id: detect-changes
+        name: Detect changes
+        uses: hawk-ai-aml/github-actions/repository-detect-changes@master
+        with:
+          repository: hawk-ai-aml/${{ inputs.component }}
+          repository-ref: ${{ env.GITHUB_REF_NAME }}
+          repository-user: ${{ secrets.RECREATE_DEVELOP_GITHUB_USER }}
+          repository-access-token: ${{ secrets.REPO_ACCESS_PAT }}
+          modules-string: ${{ steps.init.outputs.kustomize-modules-string }}
```

Comment:
I think it's better to rename them to kustomize-modules-string, like you did in line 43. The contents of the variables are the same, but they have different names

Reply:
Also does it mean that the developers have to write also a string with all the modules like it was in the old version of the multi python action?

Reply:
I don't want to rename modules-string to kustomize-modules-string because the repository-detect-changes is an independent action, and it doesn't have to understand kustomize :)

Reply:
> I think it's better to rename them to kustomize-modules-string, like you did in line 43. The contents of the variables are the same, but they have different names

Response:
I don't want to rename modules-string to kustomize-modules-string because the repository-detect-changes is an independent action, and it doesn't have to understand kustomize :)

Reply:
> Also does it mean that the developers have to write also a string with all the modules like it was in the old version of the multi python action?

Response:
The value of kustomize-modules-string will automatic calculated from profile.json